zebra/zebra_pbr.c

```diff
@@ -1044,9 +1082,9 @@ void zebra_pbr_show_ipset_list(struct vty *vty, char *ipsetname)
 			vty_out(vty, "No IPset %s found\n", ipsetname);
 			return;
 		}
-		vty_out(vty, "IPset %s type %s\n", ipsetname,
-			zebra_pbr_ipset_type2str(zpi->type));
-
+		vty_out(vty, "IPset %s type %s family %s\n", ipsetname,
+			zebra_pbr_ipset_type2str(zpi->type),
+			zpi->family == AF_INET ? "AF_INET" : "AF_INET6");
```

Comment:
isn't it more understandable for the end-user using `family2str` output here?

Reply:
ok

Reply:
still? :)

Reply:
won't be changed here?

Reply:
right.
ok